bot/src/bot.js

```diff
@@ -48,9 +67,15 @@ export function createBot(url, name, db) {
 
     const text = message.slice(name.length + 2);
 
-    saveUpdate(db, author, channel, text).then(
-      () => client.say(channel, `${author}: ${randElem(CONFIRMATIONS)}`)
-    );
+    if (text.includes('create a report for')) {
```

Comment:
hmm, I think I'd like to keep it simple for now, with just a single syntax.  anything that doesn't match is saved to the inbox right now.